### PR TITLE
docs: changing the variable camel cases

### PR DESCRIPTION
**Pull request by l-qing**
# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

/kind documentation

### CODE REVIEW

File: docs/variables.md [hunk 1]
```diff
@@ -85,46 +85,46 @@ For instructions on using variable substitutions see the relevant section of [th
...
 | `Task`        | `spec.sidecars[].args`                                          |
 | `Task`        | `spec.sidecars[].script`                                        |
 | `Task`        | `spec.workspaces[].mountPath`                                   |
 | `TaskRun`     | `spec.workspaces[].persistentVolumeClaim.claimName`             |
-| `TaskRun`     | `spec.workspaces[].configmap.name`                              |
+| `TaskRun`     | `spec.workspaces[].configMap.name`                              |
 | `TaskRun`     | `spec.workspaces[].secret.secretName`                           |
-| `TaskRun`     | `spec.workspaces[].projected.sources[].configMap.name`          |
 | `TaskRun`     | `spec.workspaces[].projected.sources[].secret.name`             |
+| `TaskRun`     | `spec.workspaces[].projected.sources[].configMap.name`          |
```
Comment (l-qing) [inline docs/variables.md:127]: This order is consistent with the declaration order of fields in the Go struct, making it easy to see if any fields are missing.

https://github.com/tektoncd/pipeline/blob/83c2f5b633e434eec9d8c7ee9bd5bff30e9f70fc/vendor/k8s.io/api/core/v1/types.go#L1721-L1733

**Review by JeromeJu — APPROVED** [review]

**Pull request merged**